test/helper.rb

```diff
@@ -2,15 +2,6 @@
 # Copyright (c) 2011 Evan Phoenix
 # Copyright (c) 2005 Zed A. Shaw
 
-if %w(2.2.7 2.2.8 2.2.9 2.2.10 2.3.4 2.4.1).include? RUBY_VERSION
```

Comment:
Hmmm... why though?

Reply:
Why move to lib I mean.

Reply:
I think I saw integration tests running Puma in a subprocess still encountering this issue on Ruby2.2. Plus I figured it made sense to warn anyone on relevant Ruby versions possibly affected by this issue that weren't already using the gem.
Hopefully not super important though since 2.2 support probably won't last much longer?

Reply:
Since we're doing a major release next, I'll revisit that policy. Dropping 2.2 especially would really help for SSL maintenance.